openedx/features/edly/middleware.py

```diff
@@ -27,6 +31,25 @@ def process_request(self, request):
         if request.user.is_superuser or request.user.is_staff:
             return
 
+        edly_sub_org = getattr(request.site, 'edly_sub_org_for_lms', None)
+        if edly_sub_org:
+            if not is_edly_sub_org_active(edly_sub_org):
+                logger.exception('EdlySubOrganization for site %s is disabled. ', request.site)
+                marketing_url = get_marketing_url_from_current_site_configurations()
+
+                if marketing_url:
+                    marketing_disabled_url = marketing_url + 'disabled'
```

Comment:
The disabled page will be the homepage of the site. We don't need to append anything with the root URL

Reply:
Done